src/elements/mod.rs

```diff
@@ -132,6 +132,8 @@ pub enum Error {
 	UnknownFunctionForm(u8),
 	/// Invalid varint7 (should be in -64..63 range)
 	InvalidVarInt7(u8),
+	/// Amount of code and signatures does not match
```

Comment:
"Number of", b/c i believe that `code` is meant as `code entries`, so it is countable.

Reply:
(Below, in string literal too)